BranchMonsterFactory/ExternalLibs/Branch-SDK/BNCConfig.h

```diff
@@ -1,34 +0,0 @@
-//
```

Comment:
It seems typical to remove the workspace and extra files, but leave the podfile.
That way users can install the cocoapods themselves.